gui/system/models.py

```diff
@@ -84,7 +84,8 @@ class Settings(Model):
         verbose_name=_('WebGUI HTTP -> HTTPS Redirect'),
         default=False,
         help_text=_(
-            'Redirect all incoming HTTP requests to HTTPS'
+            'Redirect all incoming HTTP requests to HTTPS. This '
+            'enables HSTS header to be used for each request enhancing security.'
```

Comment:
```suggestion
            'enable the HTTP Strict Transport Security (HSTS) header.'
```

Reply:
Suggested simplifying and combining the two sentences. It might be nice to add further clarification about using this setting:
 "User-agents without Transport Layer Security (TLS) will not be able to connect to this system."